hapi-fees/src/main/java/com/hedera/services/pricing/BaseOperationUsage.java

```diff
@@ -197,14 +199,32 @@ private UsageAccumulator feeScheduleUpdate() {
 	}
 
 	private UsageAccumulator hbarCryptoTransfer() {
-		throw new AssertionError(NOT_IMPLEMENTED);
+		final var txnUsageMeta = new BaseTransactionMeta(0, 2);
+		final var xferUsageMeta = new CryptoTransferMeta(380, 0,
+				0, 0);
+		final var into = new UsageAccumulator();
+		CRYPTO_OPS_USAGE.cryptoTransferUsage(SINGLE_SIG_USAGE, xferUsageMeta, txnUsageMeta, into);
+
+		return into;
 	}
 
 	private UsageAccumulator htsCryptoTransfer() {
-		throw new AssertionError(NOT_IMPLEMENTED);
+		final var txnUsageMeta = new BaseTransactionMeta(0, 0);
+		final var xferUsageMeta = new CryptoTransferMeta(380, 1,
+				2, 0);
+		final var into = new UsageAccumulator();
+		CRYPTO_OPS_USAGE.cryptoTransferUsage(SINGLE_SIG_USAGE, xferUsageMeta, txnUsageMeta, into);
+
+		return into;
 	}
 
 	private UsageAccumulator nftCryptoTransfer() {
-		throw new AssertionError(NOT_IMPLEMENTED);
+		final var txnUsageMeta = new BaseTransactionMeta(0, 0);
+		final var xferUsageMeta = new CryptoTransferMeta(380, 1,
+				0, 1);
+		final var into = new UsageAccumulator();
+		CRYPTO_OPS_USAGE.cryptoTransferUsage(SINGLE_SIG_USAGE, xferUsageMeta, txnUsageMeta, into);
+
+		return into;
```

Comment:
All of them could be reduced to a single if-else/switch as they share common code skeleton.

Reply:
I don't mind that, but I think this is more easier to understand what elements we needful each of the base operations.

Reply:
Sounds good.